src/uct/api/uct.h

```diff
@@ -509,6 +516,18 @@ struct uct_iface_params {
     uct_tag_unexp_eager_cb_t eager_cb;    /**< Callback for tag matching unexpected eager messages */
     void                     *rndv_arg;
     uct_tag_unexp_rndv_cb_t  rndv_cb;     /**< Callback for tag matching unexpected rndv messages */
+
+    /* These callbacks and address are only relevant for client-server
+     * connection establishment with sockaddr and are needed on the server side */
+    ucs_sock_addr_t             addr;
```

Comment:
ok but we decided to change this in the ucp api:
> since the struct is already called "sock_addr", the internal fields names can be shortened to "addr" and "addrlen", same as the parameter names for connect (2)

so which one?
I personally prefer to have the sockaddr, like i first committed, before the above change.